client/src/test/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClientTest.java

```diff
@@ -132,7 +132,7 @@ public void testVersionCache() throws Exception {
             .andReturn(id);
 
     // Expect only one call to lookup the subject (the rest should hit the cache)
-    expect(restService.lookUpSubjectVersion(anyString(), eq(subject)))
+    expect(restService.lookUpSubjectVersion(anyString(), eq(subject),eq(true)))
```

Comment:
@mageshn There were a couple of these, after https://github.com/confluentinc/schema-registry/pull/519 is merged (hopefully imminently) we'll be catching them automatically but in the meantime we'll highlight the ones we notice in the PR.